packages/fxa-content-server/app/styles/modules/_settings.scss

```diff
@@ -633,8 +633,7 @@ section.modal-panel {
 
   &[data-name='Fenix'],
   &[data-name='Android Components Reference Browser'],
-  // It's a hack, but the only OAuth client named "Firefox Sync" is on Android.
-  &[data-name='Firefox Sync'] {
+  &[data-name='Firefox Preview'] {
```

Comment:
FWIW, I don't think this style block will be correctly applied. Fenix sets a custom device name after signing in,  meaning that the name field ends up as something unpredictable like "Ryan's Nightly on Samsung Galaxy S8" rather than the configured OAuth client name. But that's a separate issue.